lib/logdna.rb

```diff
@@ -73,7 +50,7 @@ def default_opts
       }
     end
 
-    def level=(value)
+    def assign_level=(value)
```

Comment:
Undo this - I think you're misunderstanding the syntax here. This is a virtual attribute (http://ruby-doc.com/docs/ProgrammingRuby/html/tut_classes.html#UC) so that you can make this call while looking like an assignment, like `self.level = 3`.